<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-RIG0VRNQ" name="Default test" tests="4" failures="1" timestamp="7 Dec 2019 16:48:08 GMT" time="0.075" errors="0">
  <testcase name="enterLetter" time="0.025" classname="com.test.ConsoleInputTest">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="5">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 5
	at com.ooad.ConsoleInput.findEmptyPosition(ConsoleInput.java:99)
	at com.ooad.ConsoleInput.enterLetter(ConsoleInput.java:20)
	at com.test.ConsoleInputTest.enterLetter(ConsoleInputTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- enterLetter -->
  <testcase name="findEmptyPosition" time="0.008" classname="com.test.ConsoleInputTest"/>
  <testcase name="inEnteredLetters" time="0.001" classname="com.test.ConsoleInputTest"/>
  <testcase name="printWord" time="0.003" classname="com.test.ConsoleInputTest"/>
</testsuite> <!-- Default test -->
